dfn_configs.py

```diff
@@ -0,0 +1,259 @@
+        DfnConfig(3.5, 2.5, 0.5, 12, 0.75, 0.80, 0.55, 2.70, 1.10, 'W3525D-2,WFDD-2'),
+        # Rectangular, Type 2, 3.5 x 3.0
+        DfnConfig(3.5, 3.0, 0.5, 10, 0.95, 1.00, 0.55, 2.70, 1.60, 'V3530D-1,VFED-1'),
+        DfnConfig(3.5, 3.0, 0.5, 10, 0.75, 0.80, 0.55, 2.70, 1.60, 'W3530D-1,WFED-1'),
+        DfnConfig(3.5, 3.0, 0.5, 12, 0.95, 1.00, 0.55, 2.70, 1.60, 'V3530D-2,VFED-2'),
+        DfnConfig(3.5, 3.0, 0.5, 12, 0.75, 0.80, 0.55, 2.70, 1.60, 'W3530D-2,WFED-2'),
+        # Rectangular, Type 2, 4.0 x 3.0
+        DfnConfig(4.0, 3.0, 0.5, 10, 0.95, 1.00, 0.55, 3.20, 1.60, 'V4030D-1,VGED-1'),
+        DfnConfig(4.0, 3.0, 0.5, 10, 0.75, 0.80, 0.55, 3.20, 1.60, 'W4030D-1,WGED-1'),
+        DfnConfig(4.0, 3.0, 0.5, 12, 0.95, 1.00, 0.55, 3.20, 1.60, 'V4030D-2,VGED-2'),
+        DfnConfig(4.0, 3.0, 0.5, 12, 0.75, 0.80, 0.55, 3.20, 1.60, 'W4030D-2,WGED-2'),
+        DfnConfig(4.0, 3.0, 0.5, 12, 0.95, 1.00, 0.40, 3.70, 1.80, 'V4030D-4,VGED-4', no_exp=False),    # no nominal exp_pad
+        DfnConfig(4.0, 3.0, 0.5, 12, 0.75, 0.80, 0.40, 3.70, 1.80, 'W4030D-4,WGED-4', no_exp=False),    # no nominal exp_pad
+        DfnConfig(4.0, 3.0, 0.5, 14, 0.95, 1.00, 0.55, 3.20, 1.60, 'V4030D-3,VGED-3'),
+        DfnConfig(4.0, 3.0, 0.5, 14, 0.75, 0.80, 0.55, 3.20, 1.60, 'W4030D-3,WGED-3'),
+        # Rectangular, Type 2, 5.0 x 3.0
+        DfnConfig(5.0, 3.0, 0.5, 16, 0.95, 1.00, 0.55, 4.20, 1.60, 'V5030D-1,VJED-1'),
+        DfnConfig(5.0, 3.0, 0.5, 16, 0.75, 0.80, 0.55, 4.20, 1.60, 'W5030D-1,WJED-1'),
+        DfnConfig(5.0, 3.0, 0.5, 18, 0.95, 1.00, 0.55, 4.20, 1.60, 'V5030D-2,VJED-2'),
+        DfnConfig(5.0, 3.0, 0.5, 18, 0.75, 0.80, 0.55, 4.20, 1.60, 'W5030D-2,WJED-2'),
+        # Rectangular, Type 2, 5.0 x 4.0
+        DfnConfig(5.0, 4.0, 0.5, 14, 0.95, 1.00, 0.55, 4.20, 2.60, 'V5040D-1,VJGD-1'),
+        DfnConfig(5.0, 4.0, 0.5, 14, 0.75, 0.80, 0.55, 4.20, 2.60, 'W5040D-1,WJGD-1'),
+        DfnConfig(5.0, 4.0, 0.5, 16, 0.95, 1.00, 0.55, 4.20, 2.60, 'V5040D-2,VJGD-2'),
+        DfnConfig(5.0, 4.0, 0.5, 16, 0.75, 0.80, 0.55, 4.20, 2.60, 'W5040D-2,WJGD-2'),
+        DfnConfig(5.0, 4.0, 0.5, 18, 0.95, 1.00, 0.55, 4.20, 2.60, 'V5040D-3,VJGD-3'),
+        DfnConfig(5.0, 4.0, 0.5, 18, 0.75, 0.80, 0.55, 4.20, 2.60, 'W5040D-3,WJGD-3'),
+        # Rectangular, Type 2, 6.0 x 5.0
+        DfnConfig(6.0, 5.0, 0.5, 16, 0.95, 1.00, 0.55, 4.70, 3.40, 'V6050D-1,VLJD-1', no_exp=False),    # no nominal exp_pad
+        DfnConfig(6.0, 5.0, 0.5, 16, 0.75, 0.80, 0.55, 4.70, 3.40, 'W6050D-1,WLJD-1', no_exp=False),    # no nominal exp_pad
+        DfnConfig(6.0, 5.0, 0.5, 18, 0.95, 1.00, 0.55, 4.70, 3.40, 'V6050D-2,VLJD-2', no_exp=False),    # no nominal exp_pad
+        DfnConfig(6.0, 5.0, 0.5, 18, 0.75, 0.80, 0.55, 4.70, 3.40, 'W6050D-2,WLJD-2', no_exp=False),    # no nominal exp_pad
+    ]
+
+THIRD_CONFIGS = [
+        # Sensirion SHTC-3
+        DfnConfig(2.0, 2.0, 1.0, 4, 0.75, 0.80, 0.35, 1.60, 0.70, 'SHTC-3', lead_width=0.35),
```

Comment:
The `SHTC3` doesn't have a dash in the name AFAIK.

Also, what about allowing a name prefix in the DfnConfig? For example, this one could be called "Sensirion DFNxxxxx". You could also add the manufacturer name ("sensirion") to the keywords.

Reply:
Hm, the package will go into the Sensirion library, so it should be clear that it is a Sensirion part. Shouldn't that be enough?

I added the manufacturer name to the keywords as suggested.

Reply:
Oh I wasn't aware of that, sorry. Fixed in 
00a88d4